tests/accumulate-headers-after-304.js

```diff
@@ -75,65 +71,71 @@ export default class MyTest extends Test {
             yield true;
         });
 
-        configGen.maxPrefixSize(function *(cfg) {
-            yield 8;
-            yield 32;
-            yield 64;
-            yield 128;
-            yield 2048;
+        configGen.replyHeaderMaxSize(function *() {
+            // TODO: Justify or remove each size other than 64KB
+            yield 8*1024;
+            yield 32*1024;
+            yield 64*1024; // Squid's default; String::SizeMax_ is 64*1024-1
+            yield 128*1024;
+            yield 2*1024*1024;
```

Comment:
@eduard-bagdasaryan, please help me with this TODO: The origins of these numbers go back to branch commits 1e03a81 and fc6e910. I suspect you picked all or some of these numbers for some specific reason, and I would like to document those reasons (or drop some of these variations if they are not valuable). I have merged these changes into master, but I would like to do a followup commit to address this TODO. I can do the actual code/comment changes directly in master, so you can just just respond via a GitHub comment. Thank you.